lib/vmdb/loggers.rb

```diff
@@ -54,29 +54,29 @@ def self.create_logger(log_file_name, logger_class = ManageIQ::Loggers::Base)
     end
 
     private_class_method def self.create_container_logger(log_file_name, logger_class)
-      return nil unless (logger = create_raw_container_logger)
+      return nil unless (logger = create_raw_container_logger(log_file_name))
 
       create_wrapper_logger(log_file_name, logger_class, logger)
     end
 
-    private_class_method def self.create_raw_container_logger
+    private_class_method def self.create_raw_container_logger(log_file_name)
       return unless ENV["CONTAINER"]
 
       require "manageiq/loggers/container"
-      ManageIQ::Loggers::Container.new
+      ManageIQ::Loggers::Container.new(:progname => log_file_to_prog_name(log_file_name))
     end
 
     private_class_method def self.create_journald_logger(log_file_name, logger_class)
-      return nil unless (logger = create_raw_journald_logger)
+      return nil unless (logger = create_raw_journald_logger(log_file_name))
 
       create_wrapper_logger(log_file_name, logger_class, logger)
     end
 
-    private_class_method def self.create_raw_journald_logger
+    private_class_method def self.create_raw_journald_logger(log_file_name)
       return unless MiqEnvironment::Command.supports_systemd?
 
       require "manageiq/loggers/journald"
-      ManageIQ::Loggers::Journald.new
+      ManageIQ::Loggers::Journald.new(nil, :progname => log_file_to_prog_name(log_file_name))
```

Comment:
Similarly here, we default the logdev to nil on the other side, so we don't need it here.

```suggestion
      ManageIQ::Loggers::Journald.new(:progname => log_file_to_prog_name(log_file_name))
```